ctru-rs/src/services/ndsp/mod.rs

```diff
@@ -30,6 +31,18 @@ pub enum AudioFormat {
     PCM16Stereo = ctru_sys::NDSP_FORMAT_STEREO_PCM16,
 }
 
+/// Representation of volume mix for a channel.
+/// Each member is made up of 2 values, the first is for the "left" channel, while the second is for the "right" channel.
```

Comment:
If you wanted you could have a `struct { pub left: f32, pub right: f32 }` for each of those just to be explicit. I think it's mostly self-explanatory as-is though

Reply:
This changed, review the new version.